src/vg/civcraft/mc/citadel/reinforcement/PlayerReinforcement.java

```diff
@@ -70,6 +78,15 @@ public boolean isBypassable(Player p){
 		return gp.isAccessible(type, PermissionType.BLOCKS);
 	}
 	
+	public int getDamageMultiplier(){
+		Timestamp ts = gm.getTimestamp(g.getName());
```

Comment:
Actually disregard the second bit. Carry on with the string, I suppose. 

Maybe what I should say is I'd _prefer_ that it be standardized around the group object, since we're storing that in the reinforcement class already. But what you have is fine for now.